HW1/P2/P2a.py

```diff
@@ -1,3 +1,33 @@
+# Initialize SC context
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName="Spark 1")
```

Comment:
Just for future info, you can get the total number of cpu's and pass it along to spark.  

`config = ps.SparkConf()
config = config.setMaster('local[' + str(2*mp.cpu_count()) + ']')
config = config.setAppName('P2b')

sc = ps.SparkContext(conf=config)`